antora.yml

```diff
@@ -7,4 +7,4 @@ nav:
 asciidoc:
   attributes:
     page-origin-private: false
-    neo4j-version: '5'
+    neo4j-version: '2025.01'
```

Comment:
You can remove this whole asciidoc attributes block. The `neo4j-version` attribute is not used anywhere in the docs, and `page-origin-private` is already set in the playbook`.